using-mobile-extensions/adobe-campaignclassic/README.md

```diff
@@ -55,12 +55,16 @@ The request timeout is the amount of time, in seconds, to wait for a response fr
 1. Add the Campaign Classic extension to your project using the app's Gradle file.
 
    ```java
+   implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
    implementation 'com.adobe.marketing.mobile:campaignclassic:1.+'
    ```
 
-2. Import the Campaign Classic and Lifecycle extensions in your application's main activity.
+   or you can manually include the binaries found in [acp-sdks](https://github.com/Adobe-Marketing-Cloud/acp-sdks/) repo on GitHub.
+
+2. Import the Campaign Classic, Mobile Core and Lifecycle extensions in your application's main activity.
```

Comment:
Missing the Oxford comma - I'd go with "..., Mobile Core, and Lifecycle extensions in your..."